src/impi/core.cljs

```diff
@@ -126,6 +126,33 @@
         [w h] (:pixi.render-texture/size texture)]
     (.create js/PIXI.RenderTexture w h mode)))
 
+(defmulti draw-shape! (fn [_ v] (v :pixi.shape/type)))
+
+(defmethod draw-shape! :pixi.shape/circle 
```

Comment:
Maybe this should be `:pixi.shape.type/circle`, because that would match the key `:pixi.shape/type`. It's a little more verbose, but makes for a more consistent API since every other key works that way.